applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -728,7 +766,7 @@ See :ref:`app_build_file_suffixes` and :ref:`cmake_options` for more information
 
 .. _matter_bridge_smart_plug_functionality:
 
-Configure the functionality of the Matter-Bridge device
+Configure the functionality of the Matter bridge device
```

Comment:
Could you fix the link [here ](https://github.com/nrfconnect/sdk-nrf/pull/25247/files?file-filters%5B%5D=.rst&file-filters%5B%5D=No+extension&show-viewed-files=true#diff-f250f5581599929b90c895f576af0b1ccf1b6dc333c1d6a859ac992cae861eb9L781) also (line 781): see \`matter_bridge_app_overview\_\` -> see \`matter_bridge_app_overview\`\_